### PR TITLE
Extern dpdk

### PR DESCRIPTION
**Pull request by outscale-avu**

### CODE REVIEW

**Comment by outscale-mgo** [comment]
Can one of the admins verify this patch?

**Comment by outscale-avu** [comment]
Might be gave up because of compatibility problems, maybe

File: .travis.yml [hunk 1]
```diff
@@ -14,16 +14,15 @@ before_install:
...
- - make EXTRA_CFLAGS='-fPIC'
-
 script:
+ - mkdir -p ~/dpdk
+ - mv ./dpdk.tar.bz2 ~/dpdk
+ - cd ~/dpdk
+ - tar -xvf dpdk.tar.bz2
+ - ls ~
+ - ls ~/dpdk
+ - ls ~/dpdk/build
```
Comment (outscale-gle) [inline .travis.yml]: what ? Why did you use `ls` in this script ?
Also, you would prefer clone/copy dpdk in `$TRAVIS_BUILD_DIR/dpdk` instead of `~/dpdk`

Comment (outscale-avu) [reply .travis.yml]: ls was for testing, should be removed in the end

**Pull request closed without merging**